Framework/DataObjects/src/EventList.cpp

```diff
@@ -2220,6 +2220,47 @@ void EventList::generateCountsHistogramPulseTime(const MantidVec &X,
     }
   } // end if (there are any events to histogram)
 }
+/** With respect to PulseTime Fill a histogram given homoheneopus histogram
+*   bounds.
+* Number of bins is equal to number of elements in vector Y.
+* Appends values to existing Y values.
+*
+* @param xMin :: Minimal Pulse time (in nanoseconds,
+*                i.e. DateTime->totalNanoseconds()) value to include
+*                in binning.
+* @param xMax :: Maximal Pulse time value to constrain binning by (include the
+*                times smaller than right boundary, excluding equal)
+* @param Y :: The generated counts histogram
+* @param TOF_min -- min TOF used to constrain histogram ranges
+* @param TOF_max -- max TOF used to constrain histogram ranges
+*/
+void EventList::generateCountsHistogramPulseTime(const double &xMin,
+                                                 const double &xMax,
+                                                 MantidVec &Y,
```

Comment:
just because was just passing DateAndTime this way. I think, compiler well knows what to do in this case.